atlasdb-workload-server/src/test/java/com/palantir/atlasdb/workload/workflow/TransientRowsWorkflowsTest.java

```diff
@@ -326,23 +309,4 @@ private static WitnessedTransactionAction rewriteReadHistoryAsAlwaysInconsistent
             return action;
         }
     }
-
-    private static final class OnceSettableAtomicReference<T> {
```

Comment:
thinking out loud: nice to see how that simplified the structure of a bunch of these tests.